database/20220615_000000_Add_UserAccount_SoftcorePoints.sql

```diff
@@ -1 +0,0 @@
-ALTER TABLE UserAccounts ADD COLUMN RASoftcorePoints INT DEFAULT 0 COMMENT 'Softcore Gamerscore' AFTER RAPoints;
```

Comment:
When starting with a fresh setup and migrating the database it would create the legacy tables from https://github.com/RetroAchievements/RAWeb/blob/v2/database/migrations/legacy/2012_10_03_133633_RA_Template.sql
I moved all the recent changes into that single sql file.
Starting with the previous development dump does not work anymore as it's wildly outdated. I'll generate a new dump as soon as the bigger setup issues have been resolved. The other migration sql scripts should not be necessary anymore.
That said, importing a prod db dump should not be necessary to run the application locally, the seeders should take care of that.
What I'd like to see as an alternative is that a local instance can pull data from prod via the web api instead to seed the local instance. Dumping and cleaning prod db seems a bit extreme to me - but it will be done soon. It's still on my list.

Reply:
> until then, shouldn't these stick around

you're right, they should. I went ahead and rewrote the sql files to a single sequential migration.
this should allow to have the tables all up to date no matter when the last sql file was executed.
it will not include cleanup sql statements however (points recalc, setting timestamps, etc) - those should not be done in schema migrations.
the file will follow in one of the next commits.

Reply:
Done in https://github.com/RetroAchievements/RAWeb/pull/1103/commits/9c40f9cbdb3801188e20f438ce78e1bc35ba0ac3